js/global.js

```diff
@@ -1,27 +1,57 @@
         } else {
             pageUrl = '/';
         }
 
-        //  NAV LINKS
-        let apndNavItems = $('.navbar-items').find('a').first();
+        // INSERT NAV HTML
+        let body = document.querySelector('body'),
+            nav = document.createElement('nav');
 
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'profiles').html('Students');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'projects').html('Projects');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'events').html('Events');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'visits').html('Visits');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'about').html('About');
-        apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'contact').html('<i class=\"fas fa-envelope\"></i>');
+        nav.setAttribute('class', 'navbar');
+        nav.innerHTML = `
+            <div class="navbar-logo">
+                <a href="http://computingforall.org" class="cfa-logo"></a>
+                <a href="http://dsfastudents.org/">
+                    <img src="images/logo-color.png" />
+                </a>
+            </div>
+            <div>
+                <div class="navbar-items">
+                    <div class="hamburger">
+                        <i class="fas fa-bars"></i>
+                    </div>
+                    <div class="navbar-list">
+                        <a href="` + pageUrl + `profiles">Students</a>
+                        <a href="` + pageUrl + `projects">Projects</a>
+                        <a href="` + pageUrl + `events">Events</a>
+                        <a href="` + pageUrl + `visits">Visits</a>
+                        <a href="` + pageUrl + `about">About</a>
+                        <a href="` + pageUrl + `contact"><i class=\"fas fa-envelope\"></i></a>
+                    </div>
+                </div>
+            </div>
+        `;
 
-        $('.navbar-items').find('a').first().remove();
-        // END --  NAV LINKS
+        body.prepend(nav);
 
 
+        // let apndNavItems = $('.navbar-items').find('a').first();
+
+        // apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'profiles').html('Students');
+        // apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'projects').html('Projects');
+        // apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'events').html('Events');
+        // apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'visits').html('Visits');
+        // apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'about').html('About');
+        // apndNavItems.clone().appendTo('.navbar-list').attr('href', pageUrl + 'contact').html('<i class=\"fas fa-envelope\"></i>');
+
+        // $('.navbar-items').find('a').first().remove();
+        // // END --  NAV LINKS
```

Comment:
i'd need to delete this eventually...

Reply:
Inserting the navbar with JS is definitely a good idea, keeps the code DRY. We'll only have to refactor this huge set of pages once. Go for it. 👍 

I want those absolute paths to dsfastudent.org changed to relative paths _at some point_. It'd be a good idea to do it now, since you're touching every html file on the site already. If that's too much work we can do it in a separate PR at a separate time.

Reply:
I thought you were referring to that general problem when you said this: 

> the current code using pageUrl does an absolute / link that doesn't work on localhost

Now I understand you were specifically referring to lines 4 to 10 in js/global.js.